ted/src/main/java/frc/robot/subsystems/TofSubsystem.java

```diff
@@ -26,10 +25,12 @@ public class TofSubsystem extends SubsystemBase {
   private int canID;
```

Comment:
I suspect line 24 is wrong.  This will tend to coalesce all instances of TofSubsystem class to use a single TimeOfFlight instance.  Essentially, last TOF sensor instantiated will be the only one actually being read.  Underneath that means all three that are being used will only be reading one of the sensors.